Packs/FireEyeETP/Integrations/FireEyeETPEventCollector/FireEyeETPEventCollector.py

```diff
@@ -105,12 +108,16 @@ def get_activity_log(self, from_LastModifiedOn: str, size: int, to_LastModifiedO
         time = {"from": from_LastModifiedOn}
         if to_LastModifiedOn:
             time["to"] = to_LastModifiedOn
+        else:
+            demisto.debug(f"{LOG_LINE} empty to_LastModifiedOn. Setting to tomorrow's date")
+            time_tomorrow = datetime.now(timezone.utc) + timedelta(days=1)  # Add 1 day as a margin of error
+            time["to"] = datetime.strftime(time_tomorrow, ACTIVITY_LOG_DATE_FORMAT)
 
         req_body = assign_params(
             size=size,
             attributes=assign_params(time=time),
         )
-
+        demisto.debug(f"{LOG_LINE} request sent: {size=}, {time=}, {req_body=} ")
```

Comment:
This might be a lot of logs creating a slow down, worth consider the value vs cost,